app/models/accounting/quickbooks/connector.rb

```diff
@@ -0,0 +1,71 @@
+module Accounting
+  module Quickbooks
+    class Connector
```

Comment:
Seems like this should be called `Connection`? At first I thought it actually did some calling out to the QB api but it actually doesn't? Really it just stores the tokens and etc.?